docs/examples/configs/fluorescence.yml

```diff
@@ -7,6 +7,7 @@ fluorescence:
     yx_pixel_size: 0.325
     z_pixel_size: 2.0
     z_padding: 0
+    z_focus_offset: 0
```

Comment:
Is fluorescence changing too?

Reply:
We may need a fluorescence 3D-to-2D reconstruction as well, so I put this here. A 2D fluorescence reconstruction currently gives a `NotImplementedError`, so I think it's okay.